packages/styletron-react/src/index.js

```diff
@@ -472,6 +532,32 @@ export function createStyledElementComponent(styletron: Styletron) {
             elementProps.className = joined;
           }
 
+          if (__BROWSER__ && __DEV__ && window.__STYLETRON_DEVTOOLS__) {
```

Comment:
Same here, I think moving this into a separate function will make this easier to understand.